dp3t-sdk/sdk/src/main/java/org/dpppt/android/sdk/backend/SignatureException.java

```diff
@@ -10,7 +10,9 @@
 
 package org.dpppt.android.sdk.backend;
 
-public class SignatureException extends RuntimeException {
+import java.io.IOException;
+
+public class SignatureException extends IOException {
```

Comment:
Just a heads-up: There are `catch`-chains in the app that handle `SignatureException` as well as `IOException`.